.gitlab-ci.yml

```
@@ -189,6 +199,9 @@ build:packages:cross:
       - output/commercial
       - output/opensource
   parallel: !reference [.mender-dist-packages-image-matrix-cross, parallel]
+  retry:
+    max: 2
+    exit_codes: 192
```

Also the 137 exit code
```suggestion
  retry: 
    max: 2
    exit_codes: 
      - 137
      - 192
```

---

fixed!
and thanks!